src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -35,6 +35,15 @@
     private $end;
     private $state;
     private $values;
+    private $enableOverride;
+
+    /**
+     * @param bool $enableOverride true if you want to allow env vars overriding
```

Comment:
I'd propose to remove this comment and name the option `$overrideExistingVars` (default false) instead.

Reply:
That would be a BC break